src/formula/string_utils.hpp

```diff
@@ -131,17 +133,13 @@ std::string vngettext_impl(const char* domain,
 	vngettext_impl(GETTEXT_DOMAIN, msgid, msgid_plural, count, __VA_ARGS__)
 
 /**
- * Approximately calculates the distance between two strings
+ * @brief Calculate the Levenshtein distance of two strings.
```

Comment:
Is this correct? I thought it wasn't Levenshtein distance but a slightly different algorithm.

Reply:
The original says it was inspired by the Levenshtein distance, but made simpler for performance reasons. I figured, if I could make it fast, using a more accurate Levenshtein distance (limited, as I mention in the note) would be an improvement.